lib/languages/raku.txt

```diff
@@ -0,0 +1,10 @@
+!e
+#`(+\j-->>)+\pj
+#`{+\j-->>}+\pj
+#`[+\j-->>]+\pj
+#`(\j-->>)\pj
+#`{\j-->>}\pj
+#`[\j-->>]\pj
```

Comment:
I'm not quite sure how the `+` works with the matcher. I could be wrong but my suspicion is that these would behave the same as the above? Ignore this if my assumption is incorrect.